packages/build/config/.mocharc.json

```diff
@@ -1,7 +1,8 @@
 {
-  "require": ["source-map-support/register"],
+  "require": [
+    "source-map-support/register"
+  ],
   "recursive": true,
   "exit": true,
   "reporter": "dot"
 }
```

Comment:
Let's remove `.mocharc.json` to favor `.mocharc.js` and avoid confusions.

Reply:
I tried but it fails [here](https://github.com/strongloop/loopback-next/blob/master/packages/build/test/integration/scripts.integration.js#L315).

I think these tests check if `.mocharc.json` is copied to a new generated project, so my guess it's copied to new project.
I can changes these tests to check if `.mocharc.js` is copied correctly, but if I'm right than this will impact users because the generated mocha config file will be different.

I might be completely wrong, need your help here 😄

Reply:
I think [packages/build/bin/run-mocha.js](https://github.com/strongloop/loopback-next/blob/master/packages/build/bin/run-mocha.js#L37) uses the `.mocharc.json` as default mocha configuration file when running `lb-mocha`.

Maybe if I change it here and in the test in the previous link it will work

Reply:
We can replace all occurrences of `.mocharc.json` with `.mocharc.js`.